docs/src/modules/components/WhatsNewLayout.js

```diff
@@ -17,6 +17,102 @@ import TimelineDot from '@mui/lab/TimelineDot';
 import TimelineOppositeContent from '@mui/lab/TimelineOppositeContent';
 
 const entries = [
+  {
+    title: 'MUI X v8',
+    description: 'Cutting-edge features, enhanced customization, and full support for native ESM.',
+    date: new Date(2025, 3, 17),
+    url: 'https://mui.com/blog/mui-x-v8/',
+    highlightList: [
+      {
+        title: 'Data Grid – Pivoting [Premium]',
+        url: 'https://mui.com/blog/mui-x-v8/#pivoting',
+      },
+      {
+        title: 'Data Grid – Ask Your Table (AI Assistance) [Premium]',
+        url: 'https://mui.com/blog/mui-x-v8/#ask-your-table-ai-assistance-in-the-data-grid',
+      },
+      {
+        title: 'Data Grid – Server-Side Aggregation & Lazy Loading [Premium/Pro]',
+        url: 'https://mui.com/blog/mui-x-v8/#server-side-aggregation-and-lazy-loading',
+      },
+      {
+        title: 'Data Grid – Data Source with Support for Editing',
+        url: 'https://mui.com/blog/mui-x-v8/#data-source-with-support-for-editing',
+      },
+      {
+        title: 'Data Grid – Data Source on the Community Plan',
+        url: 'https://mui.com/blog/mui-x-v8/#data-source-on-the-community-plan',
+      },
+      {
+        title: 'Data Grid – New Toolbar',
+        url: 'https://mui.com/blog/mui-x-v8/#new-toolbar',
+      },
+      {
+        title: 'Data Grid – No Columns Overlay',
+        url: 'https://mui.com/blog/mui-x-v8/#no-columns-overlay',
+      },
+      {
+        title: 'Date & Time Pickers – Time Range Picker [Pro]',
+        url: 'https://mui.com/blog/mui-x-v8/#time-range-picker',
+      },
+      {
+        title: 'Date & Time Pickers – New View-Switching Strategy',
+        url: 'https://mui.com/blog/mui-x-v8/#new-view-switching-strategy',
+      },
+      {
+        title: 'Date & Time Pickers – Clear `ownerState` for Slots',
```

Comment:
"``" are not parsed into `pre` in this case.
```suggestion
        title: 'Date & Time Pickers – Clear "ownerState" for slots',
```